nafas/params.py

```diff
@@ -475,3 +477,15 @@
     "Sustain": "sustain.wav",
     "End": "well_done.wav",
 }
+
+CONFIG_VALIDATION_MAP = {
+    "name": str,
+    "ratio": {
+        "inhale": (int, float),
+        "exhale": (int, float),
+        "retain": (int, float),
+        "sustain": (int, float)},
```

Comment:
```
"sustain": (int, float),
},
```

Reply:
fixed in https://github.com/sepandhaghighi/nafas/pull/129/commits/8cd7044e6a7899e46dd4dbe946bf456077308f9d